src/modules/proto_dhcp/dhcpclient.mk

```diff
@@ -1,5 +1,5 @@
 TARGET		:= dhcpclient
 SOURCES		:= dhcpclient.c dhcp.c
 
-TGT_PREREQS	:= libfreeradius-radius.a
-TGT_LDLIBS	:= $(LIBS)
+TGT_PREREQS	:= libfreeradius-radius.a libfreeradius-server.a
```

Comment:
why does dhcpclient now have to link to libfreeradius-server?

The idea of `dhcp.c` was that it was code which could work "as is" in either the client or the server.

I think the correct fix here is just to update the `WARN` message, and leave the rest of the headers / linking alone.